features/zoek-met-burgerservicenummer/laatst-uitgegeven-reisdocumenten.feature

```diff
@@ -0,0 +1,122 @@
+      En de persoon heeft een 'reisdocument' met de volgende gegevens
+      | naam                                           | waarde    |
+      | soort reisdocument (35.10)                     | NI        |
+      | nummer reisdocument (35.20)                    | 3DE04ID96 |
+      | datum uitgifte Nederlands reisdocument (35.30) | 20060628  |
+      | datum einde geldigheid reisdocument (35.50)    | 20210628  |
+      Als reisdocumenten wordt gezocht met de volgende parameters
+      | naam                | waarde                     |
+      | type                | ZoekMetBurgerservicenummer |
+      | burgerservicenummer | 000000152                  |
+      | fields              | reisdocumentnummer         |
+      Dan heeft de response een 'reisdocument' met de volgende gegevens
+      | naam               | waarde    |
+      | reisdocumentnummer | NWE45TN71 |
+      En heeft de response een 'reisdocument' met de volgende gegevens
+      | naam               | waarde    |
+      | reisdocumentnummer | 3DE04ID96 |
+
+  Rule: bij zoeken van reisdocumenten met burgerservicenummer wordt een ingehouden of vermist reisdocument niet geleverd
+
+    Abstract Scenario: de persoon heeft een <omschrijving> en heeft geen ander reisdocument
+      Gegeven de persoon met burgerservicenummer '000000152' heeft een 'reisdocument' met de volgende gegevens
+      | naam                                                                    | waarde       |
+      | soort reisdocument (35.10)                                              | PN           |
+      | nummer reisdocument (35.20)                                             | NE3663258    |
+      | datum uitgifte Nederlands reisdocument (35.30)                          | 20221106     |
+      | datum einde geldigheid reisdocument (35.50)                             | 20371106     |
+      | datum inhouding dan wel vermissing Nederlands reisdocument (35.60)      | 20221229     |
+      | aanduiding inhouding dan wel vermissing Nederlands reisdocument (35.60) | <aanduiding> |
+      Als reisdocumenten wordt gezocht met de volgende parameters
+      | naam                | waarde                     |
+      | type                | ZoekMetBurgerservicenummer |
+      | burgerservicenummer | 000000152                  |
+      | fields              | reisdocumentnummer         |
+      Dan heeft de response 0 reisdocumenten
+
+      Voorbeelden:
+      | aanduiding | omschrijving                                                |
+      | I          | reisdocument ingeleverd                                     |
+      | R          | van rechtswege vervallen reisdocument                       |
+      | V          | vermist reisdocument                                        |
+      | .          | vermist of ingehouden reisdocument met onbekende aanduiding |
+
+    Scenario: de persoon heeft een reisdocument ingeleverd en heeft een ouder reisdocument van dezelfde soort nog in bezit
```

Comment:
dat kan toch niet? Het ingeleverde doc had nooit uitgegeven mogen worden zonder het vorige ongeldig te maken? Of zie ik iets over het hoofd?

Reply:
heb ik zo gedaan om het tonen van de werking. hoort niet te kunnen inderdaad

Reply:
@CathyDingemanse gaat nog eens navragen wat precies geleverd moet worden
- het laatst geleverde van elke soort reisdocument
- elk reisdocument dat nog in het bezit is van de persoon (niet ingeleverd, ingehouden, vermist, o.i.d.)
- ...

Reply:
het wordt het "elk reisdocument dat nog in het bezit is van de persoon (niet ingeleverd, ingehouden, vermist, o.i.d.)"